src/value.rs

```diff
@@ -1275,6 +1305,17 @@ mod tests {
         assert_eq!(out, timestamp);
     }
 
+    #[test]
+    #[cfg(feature = "with-chrono")]
+    fn test_chrono_utc_value() {
+        use chrono::{DateTime, NaiveDateTime, Utc};
+
+        let timestamp = DateTime::<Utc>::from_utc(NaiveDateTime::from_timestamp(61, 0), Utc);
+        let value: Value = timestamp.into();
+        let out: DateTime<FixedOffset> = value.unwrap();
+        assert_eq!(out, timestamp);
+    }
```

Comment:
This is unexpected. `DateTime<Utc>` should now be store as `Value::DateTimeUtc` but instead it's `Value::DateTimeWithTimeZone`

```rust
let out: DateTime<FixedOffset> = value.unwrap();
```

Because the following code convert all `DateTime<T>` into `Value::DateTimeWithTimeZone`

https://github.com/SeaQL/sea-query/blob/67a8ffba1351c754aae2559051930d4fe2fe70ae/src/value.rs#L295-L303

Reply:
Thanks for this observation, I had spent hours trying to find this piece of code

Reply:
Hahaaa :P